HW_1/task_2.py

```diff
@@ -0,0 +1,14 @@
+#Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали
+#одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+#*Пример:*
+#6 -> 1  4  1
+#24 -> 4  16  4'''
+
+S = int(input("Введите число: "))
+if S % 6 == 0:
+    Катя = int((S / 3) * 2)
+    Петя = int((S / 3) / 2)
+    Сережа = int((S / 3) / 2)
+    print(Петя, Катя, Сережа)
```

Comment:
Запрещено использовать кириллические символы в именах переменных и прочих объектах.
Нb функция `int`, ни скобки здесь не нужны.
```suggestion
    kate = S // 3 * 2)
    peter = sergey  = S // 3 // 2
    print(peter, kate, sergey )
```